pkg/pointerdb/metainfo/client/client.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package metainfoclient
+
+import (
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/transport"
+)
+
+var (
+	// ClientError wraps errors returned from client package
+	ClientError = errs.Class("metainfo client error")
+)
+
+// SatelliteClient maintains variables required for talking to basic satellite endpoints
+type SatelliteClient struct {
+	log       *zap.Logger
+	transport transport.Client
+	satellite *pb.Node
+}
+
+// New creates an Satellite Client
+func New(log *zap.Logger, transport transport.Client, satellite *pb.Node) *SatelliteClient {
+	return &SatelliteClient{log: log, transport: transport, satellite: satellite}
```

Comment:
Uplink doesn't know the node info, only the address. The signature should probably be:

```
func Dial(ctx context.Context, log *zap.Logger, transport transport.Client, satelliteAddress string) *Client {
```

Reply:
done